pycm/pycm_obj.py

```diff
@@ -215,23 +215,23 @@ def save_stat(
             summary=False,
             sparse=False):
         """
-        Save ConfusionMatrix in .pycm (flat file format).
+        Save the ConfusionMatrix object in .pycm (flat file format).
 
         :param name: filename
         :type name : str
         :param address: flag for address return
         :type address : bool
-        :param overall_param : overall parameters list for save, Example : ["Kappa","Scott PI]
+        :param overall_param : overall parameters list for save, Example : ["Kappa", "Scott PI"]
         :type overall_param : list
-        :param class_param : class parameters list for save, Example : ["TPR","TNR","AUC"]
+        :param class_param : class parameters list for save, Example : ["TPR", "TNR", "AUC"]
```

Comment:
`:param class_param: class parameters list for save, Example : ["TPR", "TNR", "AUC"]`